### ISSUE TITLE
Create a podcast promo component

### ISSUE THREAD

**Issue by jroebu14** [BBC-archive/psammead#3987]
**Is your feature request related to a problem? Please describe.**
We need to promote our podcast content.

**Describe the solution you'd like**
A podcast promo component to display on our pages.

View the designs here - https://zpl.io/awvWplJ

### Acceptance criteria

**Wrapper**
- should render the wrapping section element with role=region attribute
- should render the section header/label

**Title**
- should render the title text in a h3 element
- screenreaders should read out "[Brand Title], Episodes" (using visually hidden text)
- should render the link inside the h3 element and should wrap the title text
- link element should use psuedo element to make full promo clickable

**Description**
- should render the description in a paragraph element

**Image**
- should render the brand image in a figure element
- should render the brand image alt text
- should be a responsive image using correct srcset and sizes attributes
- image should lazy load and render without JS
- image should have some placeholder to prevent content shifting

**Call to action**
- should render the "Episodes" call to action in a paragraph element
- call to action should be hidden to screenreaders

**SVGs**
- Icons should be SVG and be added to psammead-assets
- SVGs should change colour when user specifies colour preferences
- SVGs should use tabindex attribute to ensure the icon is not focusable in the tabbing order (IE 11)

**Questions**
- some Qs around whether this is truncated on smaller devices or uses different data values (medium/short) that are hidden based on screen size
- Ask Ed for SVG assets then attach here

**Jira ticket** - https://jira.dev.bbc.co.uk/browse/WSMEDIA-295

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

**Comment by aeroplanejane** [comment]
Looks great! Just changed "See Episodes" to "Episodes" in the copy above and updated it in the doc here: https://paper.dropbox.com/doc/Podcast-Brand-Promo-Text--A_VZ9AV7IanPMxeC8kSk1kM5AQ-5Es8CPWRz3ItTkHwRxgOC
